src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -74,27 +85,47 @@ const WeeklySummaryOptions = props => {
         <option value="Team">Team</option>
       </select>
     </FormGroup>
-  )
-}
+  );
+};
 
 const WeeklyCommittedHours = props => {
   if (!props.canEdit) {
     return <p>{props.userProfile.weeklycommittedHours}</p>;
   }
+  const handleChange = e => {
+    // Maximum and minimum constants on lines 9 & 10
+    // Convert value from string into easy number variable
+    const value = parseInt(e.target.value);
+    if (value > MAXIMUM_WEEK_HOURS) {
+      // Check if Value is greater than total hours in one week
+      alert(`You can't commit more than ${MAXIMUM_WEEK_HOURS} hours per week.`);
+      if (value === MAXIMUM_WEEK_HOURS + 1) {
+        props.setUserProfile({ ...props.userProfile, weeklyComittedHours: MAXIMUM_WEEK_HOURS });
+        props.setChanged(true);
+      } else {
+        props.setChanged(true);
+      }
+    } else if (value < MINIMUM_WEEK_HOURS) {
+      //Check if value is less than minimum hours and set it to minimum hours if needed
+      alert(`You can't commit less than ${MINIMUM_WEEK_HOURS} hours per week.`);
+      props.setUserProfile({ ...props.userProfile, weeklyComittedHours: MINIMUM_WEEK_HOURS });
+      props.setChanged(true);
+    } else {
+      props.setUserProfile({ ...props.userProfile, weeklyComittedHours: value });
+      props.setChanged(true);
+    }
+  };
+
   return (
     <Input
       type="number"
+      min={MINIMUM_WEEK_HOURS - 1}
```

Comment:
Hi @Lunaliar - may I ask why you set min and max values this way and not just min={MINIMUM_WEEK_HOURS} max={MAXIMUM_WEEK_HOURS}?